gulpfile.js

```diff
@@ -92,19 +94,21 @@ function copyA11ySniffer () {
     .pipe(gulp.dest(`${assetsDirectory}/stylesheets/lib/`))
 }
 
-gulp.task('watch', () => {
-  gulp.watch(stylesheetsDirectory + '/**/*.scss', [ 'sass' ])
+gulp.task('watch', (done) => {
+  gulp.watch(stylesheetsDirectory + '/**/*.scss', gulp.series('sass'))
+  done()
 })
 
-gulp.task('develop', () => {
+gulp.task('develop', (done) => {
   setTimeout(() => {
     livereload.listen({
       key: fs.readFileSync(path.join(__dirname, 'src', 'main', 'resources', 'localhost-ssl', 'localhost.key'), 'utf-8'),
       cert: fs.readFileSync(path.join(__dirname, 'src', 'main', 'resources', 'localhost-ssl', 'localhost.crt'), 'utf-8'),
     })
     nodemon({
       ext: 'ts js po',
-      stdout: true
+      stdout: true,
+      ignore: ['./src/integration-test/', 'src/main/public/js/lib', 'src/test']
```

Comment:
I wonder if removing `js` from list of watched files wouldn't be better then ignoring `src/main/public/js/lib`.

Reply:
that would stop restarts on things like postcode-lookup.js.